spinta/backends/postgresql/commands/write.py

```diff
@@ -22,7 +22,11 @@ async def insert(
     table = backend.get_table(model)
     async for data in dstream:
         patch = commands.before_write(context, model, backend, data=data)
-
+        if 'coordinates' in patch.keys():
+            if 'SRID' not in patch['coordinates']:
+                coordinates = cast(sa.Column, backend.tables[model.name].columns['coordinates'])
+                srid = coordinates.type.srid
+                patch['coordinates'] = 'SRID='+str(srid)+';'+str(patch['coordinates'])
```

Comment:
This should not be gere, all geometry related code should go under geometry related command methods.